zookeeper-server/src/main/java/org/apache/zookeeper/server/util/BitMap.java

```diff
@@ -60,7 +60,7 @@ public Integer add(T value) {
             if (bit > -1) {
                 freedBitSet.clear(bit);
             } else {
-                bit = nextBit++;
+                bit = nextBit;
```

Comment:
nothing will increase nextBit then. Are you sure this is good?

Reply:
It is a dead local store error

Reply:
actualy this is not a "dead local store", because "nextBit" is a field.
tests are failing.
I will try to understand why spotbugs is concerned, eventually I will add a SuppressWarnings